mappings/package_F03/output/000163-2021/000163-2021.ttl

```diff
@@ -21,8 +21,7 @@
 <http://data.europa.eu/a4g/resource/AwardCriterionQuality/2021-S-001-000163/2172d693-1034-3515-a7ec-6f609dcc48ff>
   a epo:AwardCriterion;
   epo:hasAwardCriterionType <http://publications.europa.eu/resource/authority/award-criterion-type/quality>;
-  cccev:name """Plano de trabalhos preliminar e prazo de garantia da obra/ponderação
-                            """@pt;
+  cccev:name "Plano de trabalhos preliminar e prazo de garantia da obra/ponderação"@pt;
   cccev:weight "60"@pt .
```

Comment:
Weight should get a type (`xsd:decimal`) not a language tag. Please double check the definition in CCCEV.